### DISCUSSION TITLE
Hardhat install

### DISCUSSION THREAD

**Discussion by gotjive** [NomicFoundation/hardhat#2304] [Troubleshooting] [answered]
I make folder 📁 npm init, package.json file appears, npm install —save-dev hardhat and node_modules and package-lock.json. all good til here, then npx hardhat![image](https://user-images.githubusercontent.com/80719843/151199964-abce5ec8-5f6c-434a-836d-6bf2911bdf92.jpeg) & I get GLOBAL OPTIONS. What am I doing wrong ?

**Comment by alcuadrado** [comment, marked as answer]
Already answered on twitter: https://twitter.com/alcuadrado/status/1486393816324222983
